### ISSUE TITLE
Error when trying to export project from virtual machine

### ISSUE THREAD

**Issue by zaksnet** [godotengine/godot#33752]
**Godot version:**
Beta.1
**OS/device including version:**
Windows Server 2008
**Issue description:**
When trying to export a project from CMD using the following command `godot.exe --export --no-window` inside the project folder, i get `ALERT: Your video card driver does not support any of the supported OpenGL versions.
Please update your drivers or if you have a very old or integrated GPU upgrade it.` The message makes sense since this is a virtual machine and does not have a graphics card that supports OpenGL. What doesn't make sense is why is why the check is happening when exporting, it should only run when starting the actual game. 
I would guess that i need a headless (server) version of godot that does not have video support?

**Comment by Calinou** [comment]
Indeed, this is a duplicate of #11389. Note that you should be able to run Godot in a virtual machine by installing [mesa-dist-win](https://github.com/pal1000/mesa-dist-win), which is a software OpenGL 3.3 implementation.

**Comment by zaksnet** [comment]
> Indeed, this is a duplicate of #11389. Note that you should be able to run Godot in a virtual machine by installing [mesa-dist-win](https://github.com/pal1000/mesa-dist-win), which is a software OpenGL 3.3 implementation.

That is great, thank you! I would've never imagined such a thing existed.

**Comment by hiulit** [comment]
Just passing by to add a possible solution because I also wanted to run Godot 3.5 on a Virtual Box running Windows and I ended up using this DLL https://fdossena.com/?p=mesa/index.frag. You just have to copy it next to your Godot.exe and it works for me :)
